app/navigation/TrusteeIndividualNavigator.scala

```diff
@@ -76,7 +108,55 @@ class TrusteeIndividualNavigator extends Navigator {
         ua = ua,
         fromPage = page,
         yesCall = IDCardDetailsController.onPageLoad(index, draftId),
+        noCall = navigateToFinalPages(index, draftId, ua.is5mldEnabled)
+      )
+    case page @ LegallyIncapableYesNoPage(index) => ua =>
+      yesNoNav(
+        ua = ua,
+        fromPage = page,
+        yesCall = CheckDetailsController.onPageLoad(index, draftId),
         noCall = CheckDetailsController.onPageLoad(index, draftId)
       )
   }
+
+
+  private def navigateAwayFromDateOfBirthPages(index: Int, draftId: String, is5mldEnabled: Boolean): Call = {
+    if (is5mldEnabled) {
+      mld5Rts.CountryOfNationalityYesNoController.onPageLoad(index, draftId)
+    } else {
+      NinoYesNoController.onPageLoad(index, draftId)
+    }
+  }
+
+  private def navigateAwayFromNinoYesNoPage(index: Int, draftId: String, is5mldEnabled: Boolean, isTaxable: Boolean): Call = {
```

Comment:
`isTaxable` looks to be unused?

Reply:
It wasn't but was about to be used for the None taxable journey. It's now used